ce/api/util.py

```diff
@@ -114,4 +122,18 @@ def search_for_unique_ids(sesh, ensemble_name='ce', model='', emission='',
     if timescale:
         query = query.filter(mm.TimeSet.time_resolution == timescale)
 
+    if cell_method:
```

Comment:
the `else` branch of this, would I guess be that it doesn't filter on `cell_method`. But I think that it would be better to be explicit. Always filter on `cell_method`, but the default is `mean`. If, historically, we haven't populated the `cell_method` attribute, what if we simply populated everything with `mean` right now?

Reply:
The `cell_method` attribute is always populated.  The issue is that it hasn't been populated with the operation done to produce the climatologies.  

For example, an old `tasmax` climo file would say `time: maximum`, which describes `tasmax` itself but leaves out any further detail about what has been done with that data.  In order to describe the statistical operation it would have to say `time: maximum time: mean over days`, which is what we now do.  If we were to simply append `time: mean over days` to all the old climo files I think that would be fine.  

I agree with you though, that it would be a good idea to use `mean` as a default, since leaving it blank would give us a mix of `standard_deviation` and `mean` data.

Reply:
Yes, we need to preserve the results that existing web API clients (i.e., CE) get with their existing hits on those endpoints. If not, we have to modify all clients. Better to avoid that if possible.